examples/asr/transcribe_speech.py

```diff
@@ -161,7 +162,9 @@ class TranscriptionConfig:
     ctc_decoding: CTCDecodingConfig = CTCDecodingConfig()
 
     # Decoding strategy for RNNT models
-    rnnt_decoding: RNNTDecodingConfig = RNNTDecodingConfig(fused_batch_size=-1)
+    rnnt_decoding: RNNTDecodingConfig = RNNTDecodingConfig(
+        fused_batch_size=-1, greedy=GreedyBatchedRNNTInferConfig(use_cuda_graph_decoder=True)
```

Comment:
You can set the default config inside `GreedyBatchedRNNTInferConfig` to have this set to True by default rather than do it with this explicit override

Reply:
Same issue as with setting `use_cuda_graph_decoder=True` as default in the classes.
I added comments above this line.
`RNNTDecodingConfig` is used, e.g., in `change_vocabulary`. After this operation (useful for finetuning), the model will have the field in the config `use_cuda_graph_decoder=True`, and further training will use the CUDA decoder. Since it is not compatible with bucketing (pre-allocated memory for maximum batch_size*sequence_length can be too large in this case), I prefer to conservatively enable it only for transcription.

The alternative is to enable it everywhere by default, but in the training loop, explicitly use the decoder without CUDA graphs. However, this can make the code too complicated.

If you see that there is a more straightforward solution for this issue, please let's discuss it!

Reply:
> The alternative is to enable it everywhere by default, but in the training loop, explicitly use the decoder without CUDA graphs. However, this can make the code too complicated.
Moved to this solution. I think it is much cleaner. So, set everywhere `use_cuda_graph_decoder=True` by default